CustomScript/HandlerManifest.json

```diff
@@ -1,13 +1,17 @@
-[{
-  "name": "CustomScript",
-  "version": "1.5.1.0",
-  "handlerManifest": {
-    "installCommand": "customscript.py -install",
-    "uninstallCommand": "customscript.py -uninstall",
-    "updateCommand": "customscript.py -update",
-    "enableCommand": "customscript.py -enable",
-    "disableCommand": "customscript.py -disable",
-    "rebootAfterInstall": false,
-    "reportHeartbeat": false
+[
+  {
+    "namespace": "Microsoft.OSTCExtensions",
+    "name": "CustomScriptForLinux",
+    "extension_version": "1.5.2.0",
```

Comment:
@boumenot another place would be great. let's not introduce random fields in the schematized object and add one more place to maintain the versions. as you can see in #145 we're not particularly doing a good job maintaining version values in multiple places